checker/manual/creating-a-checker.tex

```diff
@@ -462,7 +460,6 @@ \section{Type factory: Implicit annotations\label{writing-type-introduction}}
 
 The implicit annotations may be specified declaratively and/or procedurally.
 
-
```

Comment:
It's best to have two blank lines above a section heading.  It makes the natural breaks in the document stand out even when one is reading the .tex file.